matter_server/server/device_controller.py

```diff
@@ -1256,6 +1275,13 @@ def log_node_long_setup(time_start: float) -> None:
                     )
                     # NOTE: the node will be picked up by mdns discovery automatically
                     # when it becomes available again.
+                    return
+
+                # check if this node has any custom clusters that need to be polled
+                if polled_attributes := check_polled_attributes(node_data):
+                    self._polled_attributes[node_id] = polled_attributes
```

Comment:
Should we remove the node from `self._polled_attributes` when the node goes unavailable?

Reply:
It should skip unavailable nodes already and if we remove it we need to call the expensive function again to figure out attributes to poll

Reply:
Ah I see `_custom_attributes_poller` is using `continue` when not available. Yeah make sense :+1: